pkg/cli/secretsencrypt/secrets_encrypt.go

```diff
@@ -70,11 +70,12 @@ func Enable(app *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	b, err := json.Marshal(server.EncryptionRequest{Enable: true})
+	enable := true
+	b, err := json.Marshal(server.EncryptionRequest{Enable: &enable})
```

Comment:
Just a nit, but Kubernetes has a bunch of BoolPtr stuff in their config structs so they have a utility package for it. `import "k8s.io/utils/pointer"` and do:
```suggestion
	b, err := json.Marshal(server.EncryptionRequest{Enable: pointer.BoolPtr(true)})
```

Reply:
Fixed